<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JMdictPreferences</class>
 <widget class="QWidget" name="JMdictPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="displayTab">
      <attribute name="title">
       <string>Display</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Basic information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="showJLPT">
            <property name="text">
             <string>JLPT level</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Additional information</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="lookupVerbBuddy">
            <property name="toolTip">
             <string>For transitive verbs, look for the equivalent intransitive entry and vice-versa.</string>
            </property>
            <property name="text">
             <string>Search for transitive/intransitive verb buddy</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="homophonesCount"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="toolTip">
               <string>Display words that have the same pronunciation.</string>
              </property>
              <property name="text">
               <string>Homophones</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="studiedHomophonesOnly">
              <property name="text">
               <string>Studied only</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="printingTab">
      <attribute name="title">
       <string>Printing</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="filteringTab">
      <attribute name="title">
       <string>Definitions filtering</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Filtered definitions</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Displayed:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="displayedDefs">
              <property name="toolTip">
               <string>Definitions types in this list will be searched for and displayed.</string>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="filterButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../../tagainijisho.qrc">
                <normaloff>:/images/icons/go-next.png</normaloff>:/images/icons/go-next.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="unFilterButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="../../../tagainijisho.qrc">
                <normaloff>:/images/icons/go-previous.png</normaloff>:/images/icons/go-previous.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Filtered:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="filteredDefs">
              <property name="toolTip">
               <string>Definitions types in this list will be ignored when searching and displaying entries.</string>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../tagainijisho.qrc"/>
 </resources>
 <connections/>
</ui>
